omero-ssh-systemd/Dockerfile

```diff
@@ -14,10 +14,19 @@ RUN sed -i \
 	's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' \
 	/etc/pam.d/sshd
 
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
+
 RUN useradd omero && \
 	echo 'omero:omero' | chpasswd && \
 	echo "omero ALL= (ALL) NOPASSWD: ALL" >> /etc/sudoers.d/omero
 
+RUN rm -f /lib/systemd/system/systemd*udev* ; \
+	rm -f /lib/systemd/system/getty.target;
+
+RUN yum install -y initscripts \
```

Comment:
True. On the other hand it means it's a slightly less accurate platform for testing since CentOS7 sysadmins won't have this package installed, and it means this image is becoming more tailored to our ci/test workflow instead of a general ssh/systemd container. If everyone else is happy with this then that's fine, but it'd be worth adding an explanation to the README along with a mention of the sudoers change.